contributing.md

```diff
@@ -262,6 +263,107 @@ get aquainted with this development process.
 
 1. **Done, phew.** Once you have met all the requirements, you're code will be merged. Thanks for improving Ignition Robotics!
 
+### Tracking Progress
+
+Ignition development progress is tracked publicly using a GitHub project board.
+Using project boards ensures the community has visibility to what’s coming up,
+external contributors can understand where they can help, and the reasoning behind
+development decisions are visible to everyone.
+
+The [Core development board](https://github.com/orgs/ignitionrobotics/projects/3) can be seen here.
```

Comment:
Just a personal nitpick, when I see sentences like that, I expect the link to be on the word *here*, otherwise it could be slightly confusing?

Reply:
left over habit of mine because it helps SEO haha, but that's not important for us :)

Reply:
Oh interesting, SEO is never bad. I'd be interested in hearing more about it :smile:

Reply:
I just skimmed, but I think this explains it: https://moz.com/blog/click-here-seo

I think it's a way for google to make phrase associations with the page your linking to (so in this case the SEO would benefit the core development board page, not the contributing guide). So I didn't do a great job either, the better anchor would've been `The [Ignition Core Development board]`. If anyone searches "Ignition development" or something, google will be more likely to bring that page up